lib/impress.client.js

```diff
@@ -79,44 +79,41 @@ const extCases = {
   html: (client, application) => { // eslint-disable-line
     if (typeof(client.context.data) === 'string') {
       client.end(client.context.data);
-    } else client.processingPage();
+    } else {
+      client.processingPage();
+    }
   },
 
-  sse: (client, application) => {
-    if (application.sse) {
-      client.sseConnect();
-    } else client.error(510);
+  sse: (client, application) => { // eslint-disable-line
+    if (!application.sse) return client.error(510);
+    client.sseConnect();
   },
 
   ws: (client, application) => {
-    if (application.websocket) {
-      application.websocket.finalize(client);
-    } else client.error(510);
+    if (!application.websocket) return client.error(510);
+    application.websocket.finalize(client);
   },
 
   json: (client, application) => { // eslint-disable-line
-    if (client.context.data) {
-      client.end(api.json.stringify(client.context.data));
-    } else client.error(400);
+    if (!client.context.data) return client.error(400);
+    client.end(api.json.stringify(client.context.data));
   },
 
-  jsonp: (client /*application*/) => {
-    if (client.context.data) {
-      const jsonpCallbackName = (
-        client.query.callback || client.query.jsonp || 'callback'
-      );
-      client.end(
-        jsonpCallbackName + '(' + api.json.stringify(client.context.data) + ');'
-      );
-    } else client.error(400);
+  jsonp: (client, application) => { // eslint-disable-line
```

Comment:
It's better to disable a specific rule (`no-unused-vars` in this case) than to turn off ESLint completely.